CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
-## 0.0.1+3
-
+## 0.0.1+4
 - Updates:
+- Renamed classes
```

Comment:
Hi @phil10xs, we must mention the class names changed `from` & `to` in this file.